rasa/shared/core/training_data/story_reader/yaml_story_reader.py

```
@@ -95,15 +99,8 @@ def read_from_string(self, string: Text) -> List[StoryStep]:
         Returns:
             `StoryStep`s read from `string`.
         """
-        try:
-            rasa.shared.utils.validation.validate_yaml_schema(string, CORE_SCHEMA_FILE)
-            yaml_content = rasa.shared.utils.io.read_yaml(string)
-        except (ValueError, ParserError) as e:
```

won't swallow errors anymore but instead training will fail